Misc/NEWS.d/next/Library/2017-10-05-15-14-46.bpo-31690.f0XteV.rst

```diff
@@ -0,0 +1,2 @@
+Allow to set the flags re.ASCII, re.LOCALE and re.UNICODE for the part of a
+regular expression.
```

Comment:
How about: "Allow the flags re.ASCII, re.LOCALE, and re.UNICODE to be used as inline flags for regular expressions."

Reply:
I didn't see a response.  What do you think about this suggestion?

Reply:
Sorry, I answered similar comment in other place. "inline flags" are not correct words here. All these flags already can be used as inline flags ("(?a)", "("L)", "(?u)"). But these inline flags affect the entire regular expression. What is new in this PR it is allowing them to be set only for the part of the RE: "(?a:...)" etc.

Reply:
Maybe we can call these "group flags"?  Thus:

>Allow the flags re.ASCII, re.LOCALE, and re.UNICODE to be used as group flags for regular expressions.

?

Reply:
I have never encountered such an expression, but on the other hand, there seems to be no established terminology. I'm not very like "group flags" because it contains "group", but this group is non-capturing and it is not counted in the groups returned by "match.group()" and other API. But if it looks unambiguous to you, I'll use your wording.

Reply:
The original entry felt awkwardly worded to me.  "Group flags" seems like a natural description.  Ultimately, this is just a blurb entry so it's not that critical.